csharp/Clean Code/ToDo/Program.cs

```diff
@@ -5,106 +5,131 @@ namespace ToDo
 {
     internal class Program
     {
-        public static List<string> TL { get; set; }
+        public static int option {get;set;}
+        public static List<string> TaskList { get; set; }
```

Comment:
I think that the naming convention for properties is PascalCase, so Options will be better also to maintain consistency between your 2 properties

Reply:
I appreciate you noticing. I have fixed it and I'll push it along with the unit tests.